cfme/cloud/provider/__init__.py

```diff
@@ -167,9 +167,7 @@ def _form_mapping(self, create=None, **kwargs):
 
 @NavigateWrap(Provider)
 class All(NavigateStep):
-    def prerequisite(self):
-        from fixtures.pytest_store import store
-        store.current_appliance.navigate('Dashboard')
+    prerequisite = NavigateTo('Dashboard', obj=current_appliance)
```

Comment:
I am not fond of this, one of the things I wanted to do with the new navigation was remove the need for passing context.